frontend/src/components/BreadcrumbNavigation.jsx

```diff
@@ -0,0 +1,67 @@
+import React, { useEffect, useState } from 'react';
+import Breadcrumbs from '@mui/material/Breadcrumbs';
+import { ButtonBase, Typography } from '@mui/material';
+import { useNavigation } from './NavigationContext';
+import { fetchAncestors, fetchRegion } from '../api';
+
+function BreadcrumbNavigation() {
+  const { selectedRegion, setSelectedRegion, selectedHierarchy } = useNavigation();
+  const [breadcrumbItems, setBreadcrumbItems] = useState([{ id: null, name: 'World', hasSubregions: true }]);
```

Comment:
**Suggestion:** Consider using a constant for the 'World' region object as it is used multiple times in the code.
```suggestion
const WORLD_REGION = { id: null, name: 'World', hasSubregions: true };

function BreadcrumbNavigation() {
  const { selectedRegion, setSelectedRegion, selectedHierarchy } = useNavigation();
  const [breadcrumbItems, setBreadcrumbItems] = useState([WORLD_REGION]);
  ...
}
```